cmd/soroban-cli/.soroban/identities/test_id.toml

```diff
@@ -0,0 +1 @@
+secret_key = "SBGWSG6BTNCKCOB3DIFBGCVMUPQFYPA2G4O34RMTB343OYPXU5DJDVMN"
```

Comment:
hmm. Do we really want to have a hard-coded secret key ? if the value isn't important, please dynamically generate it for testing.

Reply:
Just removed it in favor of each test getting its own temporary directory. And added a tests to write this dynamically!